### DISCUSSION TITLE
create subfolder by Multi-Rename Tool / regex

### DISCUSSION THREAD

**Discussion by hubble63** [doublecmd/doublecmd#1514] [General]
Hello,

I use MacOS.

I try to create in Replace-Command-Line a subfolder.

Depending of RexEx-Version is used, normal regex Expression is
[^/] or ^/
But both commands doesen´t work in Double Commander.
In help page of Multi-Rename Tool there is no examples for create a new folder.

I´m afraid, that DC forgot to implement the possibility to create a subfolder in the Replace-Command-Line?

Or not?

Please help.

**Comment by alexx2000** [comment]
Hi, DC does not support subfolder creation.
